Userland/Libraries/LibGfx/ImageFormats/TGALoader.cpp

```diff
@@ -87,6 +105,11 @@ static ErrorOr<void> ensure_header_validity(TGAHeader const& header, size_t whol
 ErrorOr<void> TGAImageDecoderPlugin::decode_tga_header()
 {
     m_context->header = TRY(m_context->stream.read_value<TGAHeader>());
+    if (m_context->header.id_length > 0) {
+        auto identifier = String::from_stream(m_context->stream, m_context->header.id_length);
```

Comment:
Does the spec say that this is UTF-8?

Do we want to `dbgln()` if this `is_error()`?

Reply:
The spec does not really specify the format, AFAIK it's assumed to be a string, but no specific encoding. We should probably escape non-printable/unicode characters, but this works for now. I'll post a quick follow up patch with a FIXME print.

Reply:
#25871